docs/LibcxxSupport.md

```diff
@@ -1,5 +1,8 @@
 # Open Enclave Support for libcxx
 
+The support of libcxx is based on [LLVM libc++](https://libcxx.llvm.org/),
```

Comment:
@mingweishih Can you also add couple of lines about what kind of C++ library features will likely not work within enclaves and why.  As are reader, I would find such a description very helpful in determining what parts of the library I can potentially use within an enclave.